#include "player_team_player.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

namespace playerteam{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

///<<<rpc begin
void PlayerC2GsTeamServiceImpl::EnterScene(common::EntityPtr& entity,
    const playerteam::EnterSceneRequest* request,
    playerteam::EnterSceneResponse* response)
{
///<<< BEGIN WRITING YOUR CODE EnterScene
///<<< END WRITING YOUR CODE EnterScene
}

///<<<rpc end
}// namespace playerteam
